attributes/default.rb

```diff
@@ -1,23 +1,3 @@
-#
-# Cookbook Name:: sbp_sysinternals
-# Attribute:: default
-#
-# Copyright 2014, Schuberg Philis
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-default['sysinternals']['url']               = 'https://download.sysinternals.com/files/SysinternalsSuite.zip'
-default['sysinternals']['install_dir']       = "#{ENV['SYSTEMDRIVE']}\\SysinternalsSuite"
-default['sysinternals']['bginfo_config_url'] = nil

```

Comment:
Please leave the bginfo* options in there, but commented with a remark they need to be set if they want to use it